src/common/config/config_file.cpp

```diff
@@ -441,9 +441,18 @@ bool ConfigFile::macroParse(String& value, const char* fileName) const
 			return false;
 		}
 
-		// Avoid incorrect slashes in pathnames
-		PathUtils::fixupSeparators(value.begin());
-		PathUtils::fixupSeparators(macro.begin());
+		if (flags & FORCE_SLASHES_INSTEAD_OF_BACKSLASHES)
+		{
+			size_t pos = 0;
+			while ((pos = macro.find('\\', pos)) != String::npos)
+				macro[pos] = '/';
+		}
+		else
+		{
+			// Avoid incorrect slashes in pathnames
+			PathUtils::fixupSeparators(value.begin());
+			PathUtils::fixupSeparators(macro.begin());
+		}
```

Comment:
I'm trying to keep the old implementation, so I add FORCE_SLASHES_INSTEAD_OF_BACKSLASHES flag, but maybe we can just remove it because `PathUtils::fixupSeparators();` will happen in `PATH_PARAMETER` after all parsing steps anyway?

Reply:
Please provide a sample of causing problems trace configuration and suggested patch in more traditional form (i.e. 'diff -u' acceptable by  patch utility).

Reply:
```
database = %[\\/](employee).fdb
{
	enabled = true
	log_filename = $(root)\1.log
	log_connections = true
	time_threshold = 0
}
```
Log filename should be `fb_root_path/employee.log`, but we get `fb_root_path/1.log`.

Reply:
On Windows it doesn't even get created because it's trying to parse backslashes as part of regex pattern (\1, \2, etc) but encounters \d[ev] in the file path and throws an error:
```
DESKTOP-86JTPHF	Tue Sep  3 10:37:29 2024
	Trace plugin fbtrace returned error on call trace_create.
	error while parsing trace configuration
	line 195, element "log_filename": pattern is invalid
	 E:\dev\MyFirebird\firebird\output_x64_debug\1.log
```